cpp/test/spatial/spatial_data.h

```diff
@@ -0,0 +1,1674 @@
+#include <vector>
+
+namespace raft {
+namespace spatial {
+std::vector<float> spatial_data = {

```

Comment:
Could you just add a comment saying where these data came from? If they're randomly generated, that's cool, but it would probably be good for future folks to know.